build-patched-kao-kcmo-images.sh

```diff
@@ -139,11 +139,14 @@ function update_base_image() {
     rhpkg clone containers/${brew_repo}
     pushd ${brew_repo}
     git checkout --track origin/crc-1-rhel-8
-    sed -i "s!^FROM openshift/ose-base.*!FROM $base_image!" Dockerfile
-    git add Dockerfile
-    git commit -m "Use OpenShift ${openshift_version} base image"
-    git push origin
-    rhpkg container-build
+    base_image_of_repo=$(grep "^FROM openshift/ose-base" Dockerfile | sed 's/^FROM //')
+    if [ ${base_image} != ${base_image_of_repo} ]; then
+      sed -i "s!^FROM openshift/ose-base.*!FROM $base_image!" Dockerfile
```

Comment:
you probably could do the `sed` unconditionally, and ask git if there were any changes to the work tree instead of getting `base_image_of_repo` and making a comparison.